contracts/truster/TrusterLenderPool.spec

```diff
@@ -0,0 +1,27 @@
+
+using DamnValuableToken as token;
+
+methods{
+    function token.totalSupply() external returns (uint256) envfree;
+    function token.allowance(address, address) external returns (uint256) envfree;
+    function _.execute(bytes) external => DISPATCHER(true);
```

Comment:
no need

Reply:
Done

Reply:
Not working without it , wait for @eytan-cert  response regarding the dispatch list feature

Reply:
So, after looking at the code with @nivcertora, we saw that the call's method sighash is always resolved. The dispatch list feature (unresolved call summary) only applies when the method's sighash is unresolved (`|possible_sighashes| != 1`). I believe `Dispatcher` is the appropriate summary in this case (method is resolved, contract is not)

Reply:
good to know